src/coldfront_plugin_api/scim_v2/auth_middleware.py

```diff
@@ -0,0 +1,17 @@
+import os
+import logging
+
+from django_scim.middleware import SCIMAuthCheckMiddleware
+from mozilla_django_oidc.contrib.drf import OIDCAuthentication
+
+logger = logging.getLogger(__name__)
+
+
+class SCIMColdfrontAuthCheckMiddleware(SCIMAuthCheckMiddleware):
+    def process_request(self, request):
+        if not request.user or not request.user.is_authenticated:
+            # PLUGIN_AUTH_OIDC implies DRF OIDC backend is configured
```

Comment:
This comment explains the what, rather than the why. The what can be implied by looking at the code itself. 

A better comment would say why we need to do this.